### PR TITLE
Stat 14669

### PR DESCRIPTION
**Pull request by SandeepBa-Kore**

### CODE REVIEW

File: AgentAssistAngular/src/app/components/assist/assist.component.html [hunk 1]
```diff
@@ -90,6 +90,9 @@ <h1>{{'ASSIST' | translate}}</h1>
                             ></app-feedback>
                         </ng-container>
                     </ng-container>
+                    <ng-container *ngIf="conversationSummaryObj?.editedSummary?.length !== 0">
```
Comment (saradadh-Kore) [inline AgentAssistAngular/src/app/components/assist/assist.component.html]: keep the length greater than 0.

File: AgentAssistAngular/src/app/components/assist/assist.component.ts [hunk 2]
```diff
@@ -100,6 +102,7 @@ export class AssistComponent implements OnInit, OnDestroy {
         let appState = this.localStorageService.getLocalStorageState();
         this.proactiveModeStatus = appState[this.connectionDetails.conversationId][storageConst.PROACTIVE_MODE];
         this.getInterruptDialogList();
+        this.summeryModalpopup({});
```
Comment (saradadh-Kore) [inline AgentAssistAngular/src/app/components/assist/assist.component.ts]: remove this sample summerymodalpopup function call.

File: AgentAssistAngular/src/app/components/assist/assist.component.ts [hunk 3]
```diff
@@ -497,6 +500,7 @@ export class AssistComponent implements OnInit, OnDestroy {
   }
 
   handlePopupEvent(popupObject) {
+    this.conversationSummaryObj = popupObject;
```
Comment (saradadh-Kore) [inline AgentAssistAngular/src/app/components/assist/assist.component.ts]: assign conversationSummaryObj only for summary event and on close.

Comment (SandeepBa-Kore) [reply AgentAssistAngular/src/app/components/assist/assist.component.ts]: > assign conversationSummaryObj only for summary event and on close.

Currently, we are handling the conversationSummaryObj on Submit and onClose btn click.

**Review by saradadh-Kore — APPROVED** [review]

**Pull request merged**